src/neo/Network/P2P/RemoteNode.cs

```diff
@@ -124,6 +125,12 @@ protected override void OnReceive(object message)
                     RefreshPendingKnownHashes();
                     break;
                 case Message msg:
+                    if (msg.Command == MessageCommand.Ping || msg.Command == MessageCommand.Pong)
+                    {
+                        PingPayload payload = (PingPayload)msg.Payload;
+                        if (msg.Command == MessageCommand.Ping && LastHeightSent >= payload.LastBlockIndex) break;
```

Comment:
What do you think:
- Avoid duplicate pings and pong set the `LastHeightSent` in order to avoid the duplicate ping also

Reply:
Maybe it could be be:
```cs
                        PingPayload payload = (PingPayload)msg.Payload;
                        if (LastHeightSent < payload.LastBlockIndex)  LastHeightSent = payload.LastBlockIndex;
                        if (LastHeightSent >= payload.LastBlockIndex) break;                       
```